### PR TITLE
Update and complete `acrn-dm` parameters description

### PR DESCRIPTION
**Pull request by byndhorizon**
Update and complete `acrn-dm` parameters description in the user guide and HLD

Tracked-On: #2036
Signed-off-by: Shuang Zheng shuang.zheng@intel.com

### CODE REVIEW

File: doc/developer-guides/hld/hld-devicemodel.rst [hunk 1]
```diff
@@ -89,8 +77,10 @@ options:
        --part_info: guest partition info file path
        --enable_trusty: enable trusty for guest
        --ptdev_no_reset: disable reset check for ptdev
-       --intr_monitor: enable interrupt storm monitor, params:
-       		threshold/s,probe-period(s),delay_time(ms),delay_duration(ms)
+       --debugexit: enable debug exit function
+       --intr_monitor: enable interrupt storm monitor
+       --vtpm2: Virtual TPM2 args: sock_path=$PATH_OF_SWTPM_SOCKET
+            its params: threshold/s,probe-period(s),delay_time(ms),delay_duration(ms)
```
Comment (gvancuts) [inline doc/developer-guides/hld/hld-devicemodel.rst]: Are these parameters (`thresholds, probe-period, etc`) applicable to `--vtpm2` or should it be moved up one line, under `--intr_monitor`? I think aae70db6b97e92bedba6fbc3dba17a80ef400806 actually introduced that issue in the help message in the code, if confirmed, can you fix this document *and* also include the same fix in the code (`devicemodel/core/main.c`)?

File: doc/user-guides/acrn-dm-parameters.rst [hunk 2]
```diff
@@ -107,6 +115,22 @@ Here are descriptions for each of these ``acrn-dm`` command line parameters:
...
 
+   * - :kbd:`--intr_monitor`
+     - Enable interrupt strom monitor for UOS. If user want to prevent the interrupt 
+       storm from UOS, this option should be used.
+       
+       Example:: 
+
+         intr_storm_monitor="--intr_monitor 10000,10,1,100"
+
+       - ``10000``: interrupt rate larger than 10000/s will be treated as interrupt
```
Comment (gvancuts) [inline doc/user-guides/acrn-dm-parameters.rst]: Can you introduce in the list of descriptions, the same wording that is used in the help, i.e. `threshold/s,probe-period(s),delay_time(ms),delay_duration(ms)`. That would help the reader map these more easily.

File: doc/user-guides/acrn-dm-parameters.rst [hunk 3]
```diff
@@ -132,6 +156,9 @@ Here are descriptions for each of these ``acrn-dm`` command line parameters:
 
        Example: ``-p  "1:2"`` means pin the 2nd physical cpu to 1st vcpu in UOS
 
+   * - :kbd:`--part_info <part_info_name>`
```
Comment (dbkinder) [inline doc/user-guides/acrn-dm-parameters.rst]: Documentation for ``--vtpm2`` argument is missing.
Also for ``--vmcfg`` and ``--dump``.

**Comment by dbkinder** [comment]
@hwang37 These change requests need to be addressed for this doc to make it into the 1.5 release docs.

**Comment by HuiHuangShi** [comment]
2229 <br>The PR does not touch any hypervisor source file. Skip coding guideline checks.<br>

**Comment by acrnsi** [comment]
Ok to verify

**Comment by HuiHuangShi** [comment]
2229 <br>The PR does not touch any hypervisor source file. Skip coding guideline checks.<br>

**Comment by acrnsi** [comment]
Ok to verify

**Comment by HuiHuangShi** [comment]
2229 <br>The PR does not touch any hypervisor source file. Skip coding guideline checks.<br>

File: doc/user-guides/acrn-dm-parameters.rst [hunk 4]
```diff
@@ -226,6 +272,31 @@ Here are descriptions for each of these ``acrn-dm`` command line parameters:
 
        uses ``OVMF.fd`` as the OVMF image
 
+   * - :kbd:`--virtio_poll <poll_interval>`
+     - Enable virtio poll mode with poll interval xxx ns.
+
+       usage::
```
Comment (gvancuts) [inline doc/user-guides/acrn-dm-parameters.rst]:
```suggestion
       Example::
```

**Pull request merged**